tests/test_frameDict.py

```diff
@@ -60,12 +61,14 @@ def test_FrameDictOneFrameConstructor(self):
     def test_FrameDictFrameSetConstructor(self):
         frameSet = ast.FrameSet(self.frame1, self.zoomMap, self.frame2)
         frameDict = ast.FrameDict(frameSet)
-
         indata = np.array([[1.1, 2.1, 3.1], [1.2, 2.2, 3.2]])
         predictedOut = indata * self.zoom
         assert_allclose(frameDict.applyForward(indata), predictedOut)
         assert_allclose(frameDict.applyInverse(predictedOut), indata)
 
+        frameDict2 = makeFrameDict(frameSet)
+        self.assertEqual(frameDict2.getRefCount(), 1)
```

Comment:
Out of curiosity, what was the failure? Was `frameDict2.getRefCount()` 0?

Reply:
Attempting to get the refcount fails.